lib/tasks/backfill_encrypted_dependent_data.rake

```diff
@@ -0,0 +1,13 @@
+namespace :dependents do
+  desc "Backfill account number to new encrypted column"
```

Comment:
sorry, could you remind me what the **account number** in this context was again?

Reply:
ah, the description is not updated. good catch.